assets/scss/_button.scss

```diff
@@ -2,15 +2,13 @@
 
 .d-button {
   @extend .button;
```

Comment:
not on this file, but I don't know how to comment on binary files.  I don't think ccv-sketch is being used anywhere -delete? (and generally any other files not in use)